### DISCUSSION TITLE
Build description

### DISCUSSION THREAD

**Discussion by mkauli** [AmpScm/SharpSvn#47] [Q&A]
How can I build SharpSvn from scratch? I'm using Visual Studio 2022 Community Edition 64bit. Opened Visual-Studio, cloned the repository, checked out origin/main opened SharpSvn.sln and started build via menu "Build Solution".

There comes an error with the tool Po2Resx.exe, it exited with code 1.

best regards
  Martin

**Comment by rhuijben** [comment]
You need to build the imports directory first. There is some info in the readme files. And in the .GitHub directory is an yml file that scripts everything as GitHub Action, from which you may be able to copy some tricks from.
